cdap-authorization-dataset-extn/src/test/java/co/cask/cdap/security/authorization/DatasetBasedAuthorizerTest.java

```diff
@@ -119,6 +118,12 @@ public void execute(TxRunnable runnable) throws TransactionFailureException {
         }
         transactionContext.finish();
       }
+
+      @Override
+      public void execute(TxRunnable runnable) throws TransactionFailureException {
+        execute(0, runnable);
```

Comment:
A timeout of 0 will throw an exception. Ref. https://github.com/apache/incubator-tephra/commit/36c38f0c00a9a7b2aa651263abefcdd369ff308d#diff-d67c85a89f300742a40610e7ffd21058R728